src/calibre/gui2/notify.py

```diff
@@ -28,9 +29,20 @@ def __call__(self, body, summary=None, replaces_id=None, timeout=0):
         raise NotImplementedError
 
 
+def read_file_contents(path):
+	with open(path, "rb") as f:
+		return f.read()
+
+
 class DBUSNotifier(Notifier):
 
+    #XXX: Really, this should instead just send the themable icon name
+    #
+    # Doing that however, requires Calibre to first be converted to use
+    # its AppID everywhere and then we still need a fallback for portable
+    # installations.
     ICON = I('lt.png')
+    ICON_DATA = read_file_contents(ICON)
```

Comment:
```
I('lt.png', data=True)
```

And there is no need to create a module level function just to read one file at one call site.